### ISSUE TITLE
When using the pitch black theme, the context menus are also black instead of grey

### ISSUE THREAD

**Issue by blackandyellowgecko** [SimpleMobileTools/Simple-Gallery#2122]
This lack of contrast isn't great for accessibility and it creates clutter by "merging" the context menu with other elements behind it, as if the context menu isn't on top of the main UI.

Version: 6.19.6
OS: Android 11

**Comment by tibbi** [comment]
report it at https://github.com/SimpleMobileTools/General-Discussion/issues please, it affects all apps
